### PR TITLE
Fix font properties

### LINKED ISSUE #3803: Incorrect font properties: Setting & Account details
Please apply the weights, size, and colors exactly as designed: [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5faaa48ab6e7968540370da5) 

**Incorrect in Settings:** 

- [x] Section header
- [x] Item header
- [x] Item description 

Screen Shot 2021-05-28 at 18 36 18 

**Improvements in Account details:** 

- [x] Learn more in as tertiary button
- [x] Max width for description text 
- [x] Center the Account type above the secondary button (if possible) 

Screen Shot 2021-05-28 at 18 52 14

### PR DESCRIPTION
**Pull request by dimasivan**
fix #3803

### CODE REVIEW

File: packages/suite/src/components/suite/AppNavigation/index.tsx [hunk 1]
```diff
@@ -74,6 +74,9 @@ const StyledNavLink = styled.div<{ active?: boolean }>`
     border-bottom: 2px solid
         ${props => (props.active ? props => props.theme.TYPE_DARK_GREY : 'transparent')};
     margin-right: 40px;
+    &:first-child {
+        padding-left: 0;
+    }
```
Comment (tomasklim) [inline packages/suite/src/components/suite/AppNavigation/index.tsx]: Something is not correct in the navigation bar. See the screenshots

Zeplin - main header and first nav button is more on right than section header
Screenshot 2021-06-21 at 9 37 32

Your branch:
Screenshot 2021-06-21 at 9 37 38

File: packages/suite/src/components/suite/Settings/components/Section.tsx [hunk 2]
```diff
@@ -8,13 +8,13 @@ const Wrapper = styled.div`
 `;
 
 const Header = styled.div`
-    padding: 12px;
+    padding: 12px 12px 12px 0;
```
Comment (tomasklim) [inline packages/suite/src/components/suite/Settings/components/Section.tsx]: Probably more spacing is required, see screenshots.

Zeplin: 
Screenshot 2021-06-21 at 9 32 30

Your branch:
Screenshot 2021-06-21 at 9 32 41

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/Settings/components/TextColumn.tsx [hunk 3]
```diff
@@ -27,14 +27,16 @@ const Description = styled.div`
     margin-bottom: 12px;
     margin-top: 12px;
     font-size: ${variables.FONT_SIZE.TINY};
```
Comment (tomasklim) [inline packages/suite/src/components/suite/Settings/components/TextColumn.tsx]:
```suggestion
    font-size: ${variables.FONT_SIZE.SMALL};
```

According to Zeplin, this should be 14px. You can modify Cypress tests, if it is required 😉

**Comment by tomasklim** [comment]
Check how to make fixups. If you need some help, ask us.

https://docs.trezor.io/trezor-firmware/misc/review.html

Great blog about fixups is  (found by @slowbackspace):
https://filip-prochazka.com/blog/git-fixup

**Comment by tomasklim** [comment]
Please also check the padding of Cards

Zeplin:
Screenshot 2021-06-21 at 10 02 28

App:
Screenshot 2021-06-21 at 10 02 39

File: packages/suite/src/components/suite/AppNavigation/index.tsx (hunk 1 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/AppNavigation/index.tsx]: added 3px to header and 5px to button according to zeplin

File: packages/suite/src/components/suite/Settings/components/Section.tsx (hunk 2 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/Settings/components/Section.tsx]: done

File: packages/suite/src/components/suite/Settings/components/TextColumn.tsx (hunk 3 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/Settings/components/TextColumn.tsx]: It is required to modify test as bigger font also changes the height of the application log modal

File: packages/suite/src/components/suite/AppNavigation/index.tsx [hunk 4]
```diff
@@ -74,6 +74,9 @@ const StyledNavLink = styled.div<{ active?: boolean }>`
     border-bottom: 2px solid
         ${props => (props.active ? props => props.theme.TYPE_DARK_GREY : 'transparent')};
     margin-right: 40px;
+    &:first-child {
+        padding-left: 5px;
+    }
```
Comment (tomasklim) [inline packages/suite/src/components/suite/AppNavigation/index.tsx]: - Text is moved but the border is not. I think you need to put 5px left padding on the parent of this element.
- You need to remove the padding on the right.
- It looks like the Navigation button has a bigger height in your solution than the one in Zeplin (68px vs 52px).
- Gap between these navigation buttons is too big. (63px vs 44px)

Zeplin
Screenshot 2021-07-02 at 8 44 22

App
Screenshot 2021-07-02 at 8 44 28

File: packages/suite/src/components/suite/AppNavigationPanel/index.tsx [hunk 5]
```diff
@@ -38,6 +40,7 @@ const Title = styled(H1)`
     color: ${props => props.theme.TYPE_DARK_GREY};
```
Comment (tomasklim) [inline packages/suite/src/components/suite/AppNavigationPanel/index.tsx:40]: Lines 30-35

```
const BasicInfo = styled.div`
    display: flex;
    flex-direction: column;
    padding-bottom: 11px;
    border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
`;
```

(I modified padding, check that I changed it to correct value)

File: packages/suite/src/constants/suite/layout.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 export const MAX_WIDTH = '1020px';
 export const MAX_WIDTH_WALLET_CONTENT = '768px';
 export const CARD_PADDING_SIZE = '20px';
-export const CARD_PADDING_SIZE_LARGE = '26px';
+export const CARD_PADDING_SIZE_LARGE = '18px';
```
Comment (tomasklim) [inline packages/suite/src/constants/suite/layout.ts:4]: It is now okay for switch buttons (18px) but buttons and inputs have 22px. Probably there is some more padding around them.

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/AppNavigation/index.tsx (hunk 4 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/AppNavigation/index.tsx]: 89268ed3d7a8339d697098dd6d5b9f2c5ce71356

File: packages/suite/src/constants/suite/layout.ts (hunk 6 above)
Comment (dimasivan) [reply packages/suite/src/constants/suite/layout.ts:4]: this is card padding, not button padding

File: packages/suite/src/components/suite/AppNavigationPanel/index.tsx (hunk 5 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/AppNavigationPanel/index.tsx:40]: 89268ed3d7a8339d697098dd6d5b9f2c5ce71356

File: packages/suite/src/constants/suite/layout.ts (hunk 6 above)
Comment (tomasklim) [reply packages/suite/src/constants/suite/layout.ts:4]: I know. So you have to remove padding from buttons, inputs,.. in settings

Comment (dimasivan) [reply packages/suite/src/constants/suite/layout.ts:4]: 7d68717877407421c6ee6bd8f5cff4303469f0d7

File: packages/suite/src/components/suite/AppNavigation/index.tsx [hunk 7]
```diff
@@ -23,6 +23,12 @@ const Wrapper = styled.div`
         width: 0;
         height: 0;
     }
+
+    ${props =>
+        props.settingsWrapper &&
+        `
+          height: 52px;
+      `}
```
Comment (tomasklim) [inline packages/suite/src/components/suite/AppNavigation/index.tsx]: nit:
```suggestion

    ${props =>
        props.settingsWrapper &&
        css`
          height: 52px;
      `};
```

**Review by tomasklim — APPROVED** [review]
LGTM

**Pull request merged**